components/SignUp.vue

```diff
@@ -149,20 +146,14 @@ export default {
       display: inline-block;
     }
 
-    .error {
```

Comment:
Just wondering why this was removed?

Reply:
I think this is unused, there's no element in the template with a .error class that's a descendent of a <p> element.

Reply:
Okay cool, I think it might possibly be for the error message. It might be worth checking to see if we do need the class but to move it outside of the p tag?

Reply:
It probably was for the error message but it wasn't actually doing anything, instead we were getting styling from pages/index.vue which also has a `.error` class. I've since renamed the class we use in this SignUp component to be `.modal-error` anyway since I had to change the error notification color for the white modal background.